src/dealDashboard/dealVotes/dealRepresentativesVotes/dealRepresentativesVotes.html

```diff
@@ -32,19 +30,13 @@
         text="${representative.address | smallHexString}"
       >
       </etherscanlink>
-      ${getVoteSlug(deal.allVotes, representative.address) || 'hasn’t voted'}
+      ${getVoteSlug(votes, representative.address) || 'hasn’t voted'}
       <span
-        class="representative-vote ${getVoteSlug(deal.allVotes, representative.address) ? 'representative-vote-' + getVoteSlug(deal.allVotes, representative.address) : ''}"
+        class="representative-vote ${getVoteSlug(votes, representative.address) ? 'representative-vote-' + getVoteSlug(votes, representative.address) : ''}"
```

Comment:
Can you please ensure that `votes` is being properly observed when votes are changed on other browsers.  You'll need the latest code from dev for this.

Reply:
@dkent600 I just verified that the votes are being properly observed when they are changed on different browsers! Works perfectly :)